src/components/StudentTable.tsx

```diff
@@ -0,0 +1,88 @@
+import {
+  Button,
+  Grid,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from "@mui/material";
+import MoreVertIcon from "@mui/icons-material/MoreVert";
+import Paper from "@mui/material/Paper";
+import React, { useEffect } from "react";
+import {
+  IDispatchToProps,
+  IStudentRaw,
+  IStudentState,
+} from "../state/ducks/student/types";
+import moment from "moment";
+type AllProps = IDispatchToProps & IStudentState;
+function StudentTable({ fetchStudents, data }: AllProps) {
+  useEffect(() => {
+    fetchStudents();
+  }, [fetchStudents]);
+
+  const specificDate = (time: string) =>
+    moment(time).format("MMM D, YYYY [at] hh.mm A");
+
+  const colors: Record<string, string> = {
+    "A+": "#686563",
+    "A-": "#686563",
+    "B+": "#FFF7F5",
+    "B-": "#FFF7F5",
+    F: "#FF6897",
+  };
```

Comment:
create a file named data.ts
move this function to that file and import it here.

Same reason